keyboards/ergodox/readme.md

```diff
@@ -101,6 +101,16 @@ To flash with ´teensy-loader-cli´:
 
 ## ErgoDox Infinity
 
+## Linux udev rules
```

Comment:
This file has moved.

Reply:
Should I add the note to all the input club keyboard readme files?